test/rewards-test.ts

```diff
@@ -153,7 +153,8 @@ describe('CometRewards', () => {
       const { rewards } = await makeRewards({ governor, configs: [[comet, COMP]] });
       await expect(
         rewards.setRewardConfig(comet.address, COMP.address)
-      ).to.be.revertedWith(`custom error 'AlreadyConfigured("${comet.address}")`);
+        //).to.be.revertedWith(`custom error 'AlreadyConfigured("${comet.address}")`);
```

Comment:
are these commented-out lines left in intentionally?

Reply:
Yeah, since they were stronger assertions before, I left them in case we switch to `revertWithCustomError` or something which can add back those checks